main_logic/omni_realtime_client.py

```
@@ -1134,37 +1151,37 @@ async def create_response(self, instructions: str, skipped: bool = False) -> Non
         if skipped:
             self._skip_until_next_response = True
-        
+
         # Gemini 使用 send_client_content 发送文本内容
         if self._is_gemini:
             await self._create_response_gemini(instructions)
             return
 
-        if "qwen" in self._model_lower:
-            await self.update_session({"instructions": self.instructions + '\n' + instructions})
+        # 跳过空内容的发送，避免触发 API 错误
```

Ruff (0.15.10)

[warning] Comment contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF003)